### PR TITLE
git-info: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-info.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# git info
+
+> Display repository information.
```
Comment (mfrw) [inline pages/common/git-info.md:3]: Should we be specific and mention a git repo.
I understand its under git so of course it implies a git repo. but still .. thoughts ?

```suggestion
> Display Git repository information.
```

File: pages/common/git-info.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# git info
+
+> Display Git repository information.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-info>.
+
+- Display repository information about remote locations, remote and local branches, most recent commit data and `.git/config` settings:
```
Comment (marchersimon) [inline pages/common/git-info.md:7]:
```suggestion
- Display remote locations, remote and local branches, most recent commit data and `.git/config` settings:
```
I think this is equally informative. The command doesn't really print information about those things, but lists the things themselves.

File: pages/common/git-info.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+> Display Git repository information.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-info>.
+
+- Display remote locations, remote and local branches, most recent commit data and `.git/config` settings:
+
+`git info`
+
+- Display repository information about remote locations, remote and local branches and most recent commit data:
```
Comment (CleanMachine1) [inline pages/common/git-info.md:11]:
```suggestion
- Display remote locations, remote and local branches and most recent commit data:
```

**Review by mfrw — APPROVED** [review]
LGTM :)

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Pull request merged**